pyproject.toml

```diff
@@ -1,50 +1,70 @@
-[project]
+# This fork of openapi-python-client is published internally at Benchling. Changes in this
+# project file will *not* be contributed upstream.
+
```

Comment:
I would prefer if the changes to the project structure was separated to a separate PR rather than bundled with this one

Reply:
Sorry, that was a consequence of the target branch for this PR being set wrong - fixed.